src/sentry/models/notificationsetting.py

```diff
@@ -104,6 +104,10 @@ def provider_str(self) -> str:
         ),
         null=False,
     )
+    is_migrated = models.BooleanField(
+        default=False,
```

Comment:
@snigdhas this is also bad, it's going to backfill the default. Whenever you add a column, it should be nullable.

Reply:
oh jk I guess with newer PG it's ok

Reply:
but to be safe I'd make it nullable